REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6
 DiffEqBase
 Reexport
-NLsolve
+NLsolve 0.12.1
```

Comment:
The argument flip is not until v0.14.0? @pkofod

Reply:
Indeed